### PR TITLE
Post Garnett Tidy Landing Page

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

Tidy up the bundles landing page.

cc: @Amohkhan

## Changes

- Updated the introduction.
- Updated the bundles headings and CTAs.
- Updated the "why support" and "other ways" sections.

## Screenshots

landing-full-lower

landing-trimmed-lower

### CODE REVIEW

**Comment by jranks123** [comment]
The header looks like it is getting a lot taller, how does it look on mobile? Are the payment boxes still visible?

**Comment by jranks123** [comment]
Also, what is the difference between the two screenshots?

Just a note, we should make sure that we never say "contribute or subscribe" on the contribute only version of this page.

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 1]
```diff
@@ -959,6 +953,7 @@
       font-weight: 900;
       font-size: 23px;
       line-height: 26px;
+      color: #c28a2f;
```
Comment (rupertbates) [inline assets/pages/bundles-landing/bundlesLanding.scss]: Would it be worth introducing a named colour for this?

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 2]
```diff
@@ -478,12 +472,13 @@
         }
 
         .component-cta-link {
-          background-color: #fff;
-          color: gu-colour(guardian-brand);
-          border: 1px solid gu-colour(guardian-brand);
+          background-color: #0084c6;
```
Comment (rupertbates) [inline assets/pages/bundles-landing/bundlesLanding.scss]: Would it be worth introducing a named colour for this?

**Comment by svillafe** [comment]
I agree with you @rupertbates but all these changes are sort of related to Garnett, we are going to introduce a naming system in the cleanup phase post garnett.

**Comment by svillafe** [comment]
It is working OK in PROD

**Comment by Ap0c** [comment]
> I think it would be worth introducing a named colour for colours which are used frequently though

@rupertbates the Garnett palette will dropped into the `colours.scss` file soon, haven't done it up to now for a couple reasons, including the fact that the colours kept changing!

**Comment by Ap0c** [comment]
Thanks for getting this out @svillafe!

**Comment by Ap0c** [comment]
Further updates, including better colour naming, in #449.

**Pull request merged**